README.md

```diff
@@ -57,113 +69,158 @@ Subdirectory README files should follow the same general format as this root fil
-* Alice's wallet constructs the JSON request to send to the server
+### Payment Tracking
 
-```javascript
-Authorization: Bearer {jwt}
-{
-    "coin": "BTC",
-    "address": "bc1qvw0ytfntx6zs0lfsruem6xwj0mewng523ktatp",
-}
-```
+It is fairly simple for a third party to constatly poll a given alias's endpoint and record all that alias's addresses over time. While the alias protocol isn't necessarily meant to have stringent privacy measures (the whole point of an alias is to relate a public account to an address) there are a couple things that can be done to increase privacy:
 
-* Alice sends the request and her address is updated on the server
+* Coins that are able to implement features similar to BIP 47 should do so and use payment codes instead of regular addresses.
+* Servers can have sign-ups that don't require any personal information, so users can use an anonymous alias.
+
+<hr>
+
+## Justifications
+
+* **Squatting:** CAP is a REST protocol built on top of DNS to stop "squatters". Squatting is when users that are the first-adopters of the protocol come in and steal all the valuable aliases ("nike", "coke", "trump", "btc", etc). By requiring that the a domain name is part of an alias, users have to first own the domain which is a system that is already fairly distributed.
+
+* **TXT Records**: There is another protocol out there, OpenAlias, that proposes the usage of domain TXT Records for transmitting alias/address combinations. A lot of good work was done there, but we designed this system because we felt that most defvelopers are more comfortable building out a simple CRUD app with a database. If it is easier for developers to adhere tothe protocol, it will naturally be more distibuted and decentralized.
+
+* **Why no blockchain?** The amount of data being stored on blockchains is a constant struggle, we are always trying to reduce the amount of data that needs to be verified by all nodes in a decentralized network (see the bitcoin scaling problem)Namecoin also never experienced great adoption so we saw no reason to beat that dead horse.
+
+<hr>
+
+## ENDPOINTS
+
+All endpoints (Tier 1 and Tier 2) adhere to RESTful standards and communicate using JSON. 
+
+All endpoints are served at the URL in the following format:
+
+https://opencap.{domain}.{extension}:443/{endpoint}
+
+HTTPS is required for all CAP endpoints, and as is standard the port 443 will be used.
+
+Endpoints are prefixed with Protocol version in case of non-backwards compatibility.
+
+<hr>
+
+## TIER 1 ENDPOINT
 
-### GET /address/{username}/{coin}
+The following GET endpoint is the only endpoint a server needs to support to be CAP 1 compliant. This is because in order for a user to have a CAP alias, wallets just need to know where to look up the address. CAP 1 can be as simple as a static string served from the endpoint.
+
+### GET /v1/address/{ledger_id}/{username}

```

Comment:
We should swap `ledger_id` and `username`.  
Also, the word 'address' is confusing in this context. If we want to make this API RESTful it should look like `GET /v1/users/{username}/ledgers/{ledger_id}`.

Reply:
We can't in order to stay REST compliant. the DELETE /v1/address/{ledger_id} inhibits us.

According to REST the it is called /address because we are getting an address. It wouldn't be /user because we aren't doing anything with users, its just a parameter

Reply:
@FrilledShark Exactly. We still need to discuss whether we stick with the word 'Ledger ID' as we can support non-cryptocurrency projects like BitMessage as well. But on the other hand, do we really need to change the word just because there are a few exceptions?

Reply:
The database sure, but I'm just talking about resource access via the API. Naming the endpoint after the returned resource (address in our case) is standard. As for the name.... ledger is super obscure, I prefer to call them coins, and there will just be some exceptions. No big deal.